notification-portlet-webapp/build.gradle

```diff
@@ -100,11 +100,17 @@ war {
     }
 }
 
+task cleanMinifiedJavascript(type: Delete) {
+    delete fileTree(dir:'src/main/webapp/scripts', include: '**.min.js')
```

Comment:
There are source files in this location:  https://github.com/Jasig/NotificationPortlet/tree/master/notification-portlet-webapp/src/main/webapp/scripts

We need to produce files in the `build/` directory.